src/js/visual/configurator-csr.js

```diff
@@ -1025,34 +1025,37 @@ ripe.ConfiguratorCsr.prototype._initDebug = function() {
             this.debugRefs.renderedInitials.group.add(this.debugRefs.renderedInitials.axis);
         }
 
-        // inits reference points curve
-        if (this.debugOptions.renderedInitials.line) {
-            const curve = new window.THREE.CatmullRomCurve3(
-                this.initialsRefs.renderedInitials.points,
-                false,
-                "centripetal"
-            );
-            const pointsNum = 50;
-            const linePoints = curve.getPoints(pointsNum);
-            this.debugRefs.renderedInitials.line = new window.THREE.Line(
-                new window.THREE.BufferGeometry().setFromPoints(linePoints),
-                new window.THREE.LineBasicMaterial({ color: 0x0000ff })
-            );
-
-            this.debugRefs.renderedInitials.group.add(this.debugRefs.renderedInitials.line);
-        }
+        // ensures it has the minimum number of points
+        if (this.initialsRefs.renderedInitials.points.length > 1) {

```

Comment:
Ok, sorry again, but isnt the minimum 1? In other words, dont you want this to be > 0?
If it is some ad-hoc maximum value should that be hardcoded or softcoded as an argument?

Reply:
No, I told you the min and max can vary.

Reply:
No, I wan't the minimum to be 2 as you can only have a line with a minimum of 2 points.